slack_bolt/slack_function/slack_function.py

```diff
@@ -78,8 +77,95 @@ def __call__(*args, **kwargs):
 
         return __call__
 
-    # TODO add view listener
+        # -------------------------
+
+    # view
+
+    def view(
+        self,
+        constraints: Union[str, Pattern, Dict[str, Union[str, Pattern]]],
+        matchers: Optional[Sequence[Callable[..., bool]]] = None,
+        middleware: Optional[Sequence[Union[Callable, Middleware]]] = None,
+    ) -> Callable[..., Optional[Callable[..., Optional[BoltResponse]]]]:
+        """Registers a new `view_submission`/`view_closed` event listener for a function.
+
+            # Use this method as a decorator
```

Comment:
"This method" is probably a copy-paste from the other docstring for the top-level function handler, yes? Probably want to rename this to something else. Maybe:
```suggestion
            # Define a function handler using the function decorator
```

Reply:
Added this to the function.actions handler as well 🥇